network.go

```diff
@@ -575,7 +577,20 @@ func (s *sandbox) updateRoute(netHandle *netlink.Handle, route *types.Route, add
 // DNS //
 /////////
 
-func setupDNS(dns []string) error {
+func setupDNS(dns []string) (err error) {
```

Comment:
nit: So looks like you are now sending the dns string from the runtime itself. Can you mention that in the commit as well.

Reply:
I dont see where this function is being called, could you point me to it ?

Reply:
Re: sorry I did not see is a replacement the function, it is not a new called function.